app/controllers/albums.js

```diff
@@ -0,0 +1,15 @@
+const albumsService = require('../services/album');
+
+module.exports.albums = (_, res, next) => {
+  albumsService
+    .albums()
+    .then(albumsList => albumsList)
+    .catch(error => next(error));
```

Comment:
There is a syntactic sugar for this:
`.catch(next);`
Why does this work? `next` and `error => next(error)` are both uncalled functions.  And all `error => next(error)` is doing is calling `next`, so you can just send the uncalled `next` function.
If i'm not being clear about this, please ask me, its important that you understand how functions work, in order to apply more advanced functional programming patterns.